feat(dashboards): Manage dashboards update (#16900)
newrelic/docs-website: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx

* Export any dashboard as a PDF file clicking the <Icon name="fe-more-horizontal"/> icon on the right corner and select <DoNotTranslate>**Export dashboard as PDF**</DoNotTranslate>.
* Share your charts either as a PNG image or as a link: Click the <Icon name="fe-more-horizontal"/> icon on the right corner of the chart and select the <DoNotTranslate>**Get as image**</DoNotTranslate>.

Exporting to PDF or as an image has certain limitations depending on the chart type.

## Share your data [#data-share]

You may be used to any user in your organization having the ability to create live chart URLs by using the <DoNotTranslate>**Get chart link**</DoNotTranslate> option in supported visualizations. Now, in order to prevent potential data exposures, administrators will have control over who can and can't create live chart URLs. This permission is now controlled by both the <DoNotTranslate>**Live Chart URL (self)**</DoNotTranslate> and <DoNotTranslate>**Live Chart URL (all)**</DoNotTranslate> capability, explained below. Using these, you can assign permissions to accounts to enable individual users to create shareable URLS.

> In order to minimize disruption, customers who want to enable this functionality can create custom roles and assign them to their users prior to the GA date of April 30. The capabilities won't activate until the GA date.

Reviewer: In order to minimize disruption, customers who want to enable this functionality can create custom roles and assign them to their users prior to April 30, 2024. After April 30, this function will be generally available, and these capabilities will activate.

> You may be used to any user in your organization having the ability to create live chart URLs by using the <DoNotTranslate>**Get chart link**</DoNotTranslate> option in supported visualizations. Now, in order to prevent potential data exposures, administrators will have control over who can and can't create live chart URLs. This permission is now controlled by both the <DoNotTranslate>**Live Chart URL (self)**</DoNotTranslate> and <DoNotTranslate>**Live Chart URL (all)**</DoNotTranslate> capability, explained below. Using these, you can assign permissions to accounts to enable individual users to create shareable URLS.
> 
> In order to minimize disruption, customers who want to enable this functionality can create custom roles and assign them to their users prior to the GA date of April 30. The capabilities won't activate until the GA date.

Reviewer 2: ```suggestion
You may be used to any user in your organization having the ability to create live chart URLs by using the <DoNotTranslate>**Get chart link**</DoNotTranslate> option in supported visualizations. Now, to prevent potential data exposures, administrators will have control over who can and can't create live chart URLs. This permission is now controlled by both the <DoNotTranslate>**Live Chart URL (self)**</DoNotTranslate> and <DoNotTranslate>**Live Chart URL (all)**</DoNotTranslate> capability, explained below. Using these, you can assign permissions to accounts to enable individual users to create shareable URLS.
		

To minimize disruption, customers who want to enable this functionality can create custom roles and assign them to their users before the GA date of April 30. The capabilities will activate on the GA date.
```
